tests/multiparser_js_graphql/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -82,6 +78,153 @@ const query = gql\`
     }
   }
 \`;
+
+
+// With interpolations:
+
+gql\`
+query User {
+  user(id:5){
+    ...UserDetails
+    ...Friends
+  }
+}
+
+\${USER_DETAILS_FRAGMENT}\${FRIENDS_FRAGMENT}
+\`
+
+
+// Skip if non-toplevel interpolation:
+
+gql\`
+query User {
+  user(id:\${id}){ name }
+}
+\`
+
+
+// Skip if top-level interpolation within comment:
+
+gql\`
+query User {
+  user(id:5){ name }
+}
+#\${test}
+\`
+
+
+// Comment on last line:
+
+gql\`
+query User {
+  user(id:5){ name }
+}
+# comment\`
+// \` <-- editor syntax highlighting workaround
+
+
+// Preserve up to one blank line between things and enforce linebreak between
+// interpolations:
+
+gql\`
+# comment
+\${one}\${two}  \${three}
+\${four}
+
+\${five}
+# comment
+\${six}
+
+# comment
+\${seven}
+# comment
+
+\${eight}
+
+  # comment with trailing whitespace      
+
+
+# blank line above this comment
+
+
+\`
+
+
+// Interpolation directly before and after query:
+
+gql\`\${one} query Test { test }\${two}\`
+
+
+// Only interpolation:
+
+gql\`\${test}\`
+
+
+// Only comment:
+
+gql\`# comment\`
+// \` <-- editor syntax highlighting workaround
+
+
+// Only whitespace:
+
+gql\`   \`
+
+
+// Empty:
+
+gql\`\`
+
+
+// Comments after other things:
+// Currently, comments after interpolations are moved to the next line.
+// We might want to keep them on the next line in the future.
+
+gql\`
+  \${test} # comment
+
+  query Test { # comment
+    test # comment
+  } # comment
+  \${test} # comment
+  \${test} # comment
+
+  \${test} # comment
+
+  # comment
+  \${test} # comment
+\`
```

Comment:
All the comments after the interpolations here are currently moved to the next line. Is this a deal-breaker? This seems to need some larger changes to change.

Reply:
Probably not a deal-breaker. I'm guessing comments are pretty uncommon here. I've only ever used comments in GraphQL schemas.